src/components/NavBar/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import './NavBar.scss';
+import logoFull from '../../assets/logo.svg';
+
+export const NavBar = () => {
+  const [expanded, setExpanded] = useState(true);
+
+  const handleExpantion = () => {
+    setExpanded(!expanded);
+  }
+
+  const NavBarClasses = classNames('NavBar', {
+    'isContracted': !expanded,
+  })
+
+  return (
+    <nav className={NavBarClasses}>
+      <div className="NavBar__trigger" onClick={handleExpantion}>
+        {expanded ?
+          <i className="icon__arrow-narrow--left"></i> :
+          <i className="icon__arrow-narrow--right"></i>
+        }
+      </div>
+        <div className='NavBar__logo'>
+          {expanded ? <img src={logoFull} alt='Hope Tracker' /> : <i className='icon__brand'></i> }
+        </div>
+        <ul className='NavBar__list'>
+          <li className="NavBar__item">
```

Comment:
This can be a reusable component, a have a state or props in NabBar type array like `[{to:"/global", label: "Global"}, {to:"/", label: "Home"}]` and iterate with a map function and return the NavItem component